### PR TITLE
Update

### PR DESCRIPTION
**Pull request by jburel**
* Update doc
* Add entry to changelog
* Remove 5.1.x support in README and code

### CODE REVIEW

File: omero_marshal/__init__.py [hunk 1]
```diff
@@ -53,7 +53,7 @@ def get_schema_version(version):
     if m is None:
         raise Exception("Invalid OMERO version number: " + version)
     v = StrictVersion(m.group(1))
-    if v >= StrictVersion('5.1.0') and v < StrictVersion('5.3.0'):
```
Comment (sbesson) [inline omero_marshal/__init__.py]: Although OMERO 5.1 is no longer supported by the library, was there any issue associated with the mapping between the OMERO version and the underlying OME model version?

File: setup.py [hunk 2]
```diff
@@ -61,6 +61,7 @@ def read(fname):
         'Natural Language :: English',
         'Operating System :: OS Independent',
         'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 3',
```
Comment (sbesson) [inline setup.py]: Unsure whether we want to add this classifier at this stage. Although this repository itself might be Python 3 compatible, the library will remain largely non-functional on Python 3 until OMERO.py becomes Python 3 compatible.
Is there an official reference defining the meaning of this classifier so far.

Comment (jburel) [reply setup.py]: it is what you mean http://pypi.python.org/pypi?%3Aaction=list_classifiers

**Comment by joshmoore** [comment]
Will this need updating for 5.5?

**Comment by jburel** [comment]
@joshmoore yes

**Pull request merged**